### ISSUE TITLE
Issue for triggering new releases

### ISSUE THREAD

**Issue by ablaom** [JuliaAI/MLJTuning.jl#59]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] Add support for MLJ's upcoming iteration API (https://github.com/alan-turing-institute/MLJ.jl/issues/139, #116)

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] Extend compat for MLJBase to ^0.17.7, ^0.18

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- [x] (**enhancement**) Make plotting for machines bound to `TunedModel` instances work for more than two hyper-parameters (#41, PR #121)
- [x] Extend compatibility for MLJModelInterface to include version 1.0

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

cc @lhnguyen-vn

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

release notes:

- [x] (**enhancement**) Allow the use of models of varying type in the `Explicit` strategy and provide a convenient syntax `TundedModel(models=..., )` (#142)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

release notes:

- [x] Fix problem with reproducibility when `acceleration isa CPUThreads` (#126, #145) @lhnguyen-vn

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

release notes:

- [x] (**enhancement**) Allow specification of deterministic measures for probabilistic predictors (#148)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**breaking**) Change default tuning strategy from `Grid` to `RandomSearch()` (#147)
- (**breaking**) Remove deprecated `learning_curve!` method (#151)
- (**breaking**) Adapt to serialization changes in MLJBase 0.20. In particular, this allows serialization of `TunedModel(model=...)` when `model` is not pure Julia (#165) @olivierlabayle

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**enhancement**) Add user to specify class weights  when constructing `TunedModel` , for passing to measures (#172)
- (**convenience**) Allow user to specify models as an argument instead of keyword, as in `TunedModel(model, tuning=...)` (#175)
- Suppress some unecessary "outer layer" caching of training data (#173)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**breaking**) Bump MLJBase compatibility to version 1. When using without MLJ, users may need to explicitly import StatisticalMeasures.jl. See also the [MLJBase 1.0 migration guide](https://github.com/alan-turing-institute/MLJ.jl/blob/measure/docs/src/performance_measures.md#migration-guide-for-changes-to-measures-in-mljbase-10) (#194)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**breaking**) Bump MLJBase compatibility to version 1. When using without MLJ, users may need to explicitly import StatisticalMeasures.jl. See also the [MLJBase 1.0 migration guide](https://github.com/alan-turing-institute/MLJ.jl/blob/measure/docs/src/performance_measures.md#migration-guide-for-changes-to-measures-in-mljbase-10) (#194)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Include full evaluation objects (key = `:evaluation`) in history entries (#210)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**enhancement**) Implement feature importances that expose the feature importances of the optimal atomic model (#213)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Write the `PerformanceEvaluation` objects computed for each model (hyper-parameter set) to the history, or write compact versions of the same (`CompactPeformanceEvaluation` objects) by providing `TunedModel(...)` a new option `compact_history=true`.  The evaluation objects are accessed like this: `evaluation = report(mach).history[index].evaluation`, where `mach` is a machine associated with the `TunedModel` instance. For more on the differences between `PerformanceEvaluation` and `CompactPerformanceEvaluation` objects, refer to their document strings. (In MLJTuning 0.5.3 and 0.5.4 an experimental feature already introduced `PerformanceEvalution` objects to the history, but with no option to write the compact form. In the current release, compact objects are written *by default*.)

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

- (**new feature**) Add `logger` option to `TunedModel` wrapper, for logging internal model evaluations to an ML tracking platform,  such as MLflow via [MLJFlow.jl](https://github.com/JuliaAI/MLJFlow.jl). Default should be `nothing` for no logging (#193). The logger must support asynchronous messaging if `TunedModel(model, ...)` is specified with the option `acceleration=CPUThreads()` or `CPUProcesses()`. Currently, `CPU1()` (the default) is supported by MLJFlow.jl's loggers, while ansynchronous support is a work in progress; see https://github.com/JuliaAI/MLJFlow.jl/pull/41 #193.

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Change default `logger` from `nothing` to `MLJBase.default_logger()` (which can be reset with `MLJBase.default_logger(new_logger)`) #221

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Declutter the hyper-parameter labels in plots (#226)
